manager/integration/tests/test_basic.py

```diff
@@ -59,6 +63,44 @@ def test_hosts_and_settings(clients):  # NOQA
         assert setting["value"] == old_value
 
 
+def volume_read(fdev, start, count):
+    fdev.seek(start)
+    return fdev.read(count)
+
+
+def volume_write(fdev, start, data):
+    fdev.seek(start)
+    fdev.write(data)
+    fdev.flush()
+
+
+def volume_valid(dev):
+    return stat.S_ISBLK(os.stat(dev).st_mode)
+
+
+def volume_rw_test(dev, count):
+    check_count = 0
+    if not volume_valid(dev):
+        return check_count
+
+    with open(dev, "rb+") as fdev:
+        # save data
+        origin_data = volume_read(fdev, VOLUME_RWTEST_START_POSITION, count)
+
+        # rw test
+        w_data = b"\xfe"*(count)
```

Comment:
Don't write the same character. Use random string, so if there is a difference between this write and previous write, it will be detected.